generator/header_tmpl.go

```diff
@@ -9,9 +9,11 @@ package {{.}}
 // Do not modify
 // Copyright (c) 2015, Hooklift. All rights reserved.
 import (
+    "net/http"
```

Comment:
It is required for the operations https://github.com/notzippy/gowsdl/blob/add_soap_header/generator/operations_tmpl.go#L38